src/SDKs/Storage/Management.Storage/Generated/SdkInfo_StorageManagementClient.cs

```diff
@@ -0,0 +1,31 @@
+
+// <auto-generated>
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Management.Storage
+{
+  using System;
+  using System.Collections.Generic;
+  using System.Linq;
+
+  internal static partial class SdkInfo
+  {
+      public static IEnumerable<Tuple<string, string, string>> ApiInfo_StorageManagementClient
+      {
+          get
+          {
+              return new Tuple<string, string, string>[]
+              {
+                new Tuple<string, string, string>("Storage", "BlobContainers", "2018-02-01"),
```

Comment:
Please replace line 4 with `<AzureApiTag>Storage_2018-02-01;</AzureApiTag>` [here](https://github.com/blueww/azure-sdk-for-net/blob/f8971b1160bd2f19b3a1c38d007c9c149afd0d74/src/SDKs/Storage/AzSdk.RP.props)

Reply:
I have fixed it. 
I think this should be automatically updated when build. Do I miss anything to update it? Or is there any code error in SDK framework make this not update in build?

Reply:
It should be but not sure why it fails, better do this manually since this is a rush job